src/server.ts

```diff
@@ -165,6 +188,8 @@ export async function startPluginsServer(
             await stopFastifyInstance(fastifyInstance!)
         }
         await queue?.stop()
+        await pubSub?.quit()
+        await kafkaProducer?.disconnect()
         pubSub?.disconnect()
```

Comment:
any issue with `pubSub.quit()` being before `pubSub.disconnect()`?

Reply:
Ah, yep, I think disconnect should go first. Added `quit` as well because otherwise Redis connection wasn't shut down cleanly (or at all sometimes).